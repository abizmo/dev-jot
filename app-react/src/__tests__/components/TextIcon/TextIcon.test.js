import { shallow } from "enzyme";
import TextIcon from "../../../components/TextIcon/TextIcon";

describe('TextIcon tests', () => {
  test('should render and match snapshot', () => {
    const props = {
      icon: 'some icon',
      text: 'some text',
      alt: 'some description',
    };
    const wrapper = shallow(<TextIcon {...props} />);

    expect(wrapper.find('h2').text().trim()).toBe(props.text);
    expect(wrapper.find('img').prop('src')).toBe(props.icon);
    expect(wrapper).toMatchSnapshot();
  });
});
