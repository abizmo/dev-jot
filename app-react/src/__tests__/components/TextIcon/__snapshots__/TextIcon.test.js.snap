// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextIcon tests should render and match snapshot 1`] = `
<h2
  style={
    Object {
      "color": "white",
    }
  }
>
  <img
    src="some icon"
  />
   
  some text
</h2>
`;
